_cache/rack/body/69/6d474617a4e0d4abbc44db81b077efc586e2ae

```diff
@@ -0,0 +1 @@
+{"currencies":[{"code":"TWD","name":"New Taiwan dollar"},{"code":"USD","name":"United States dollar"},{"code":"EUR","name":"Euro"},{"code":"GBP","name":"Pound sterling"}]}
```

Comment:
the file under directory `_cache` shouldn't push to git repository. add _cache/* into .gitignore